### ISSUE TITLE
deleted items in search-result will be deleted, when leaving search-window -- no '$' needed

### ISSUE THREAD

**Issue by klartext** [akrennmair/newsbeuter#122]
Do a serach in a feed with '/'.
Use 'D' to delete items.
Use 'q' to quit search and go back to feed.

The deletion is done, even though '$' was not pressed.

So, the "just mark, process later" does not work.

(The deleted items are still in the feed-view, but thats another issue (just reported some minutes ago).)

**Comment by Minoru** [comment]
Moving this to 2.11 milestone to unblock 2.10 release.
